docs/source/extension/extension_points.rst

```diff
@@ -1041,6 +1041,83 @@ a plugin:
       }
     };
 
+Kernel Subshells
+----------------
+
+Kernel subshells enable concurrent code execution within kernels that support them. Subshells are separate threads of execution that allow interaction with a kernel while it's busy executing long-running code, enabling non-blocking communication and parallel execution.
+
+**Kernel Support**
+
+Subshells are supported by:
+
+- **ipykernel 7.0.0+** (Python kernels) - Kernels advertise support via ``supported_features: ['kernel subshells']`` in kernel info replies
+- Other kernels implementing `JEP 91 <https://jupyter.org/enhancement-proposals/91-kernel-subshells/kernel-subshells.html>`__
+
+**User Interface**
+
+Users can create subshell consoles in two ways:
+
+1. **Command Palette**: Press ``Ctrl+Shift+C`` and search for "New Subshell Console for Notebook"
+2. **Context Menu**: Right-click in a notebook (only visible when kernel supports subshells)
+
+To verify subshell functionality, use the ``%subshell`` magic command in Python kernels:
+
+.. code:: python
+
+  %subshell
+  # Output shows:
+  # subshell id: None (main shell) or unique ID (subshell)
+  # subshell list: ['id1', 'id2', ...] (active subshells)
```

Comment:
This is duplicated with a very similar section in user-facing docs, can you replace it with one sentence linking to `_subshell-console` target?